config/default/manager_auth_proxy_patch.yaml

```diff
@@ -11,13 +11,21 @@ spec:
       containers:
       - name: kube-rbac-proxy
         image: gcr.io/kubebuilder/kube-rbac-proxy@sha256:db06cc4c084dd0253134f156dddaaf53ef1c3fb3cc809e5d81711baa4029ea4c
+        resources:
+          limits:
+            cpu: 500m
+            memory: 128Mi
+          requests:
+            cpu: 5m
+            memory: 64Mi
         args:
         - "--secure-listen-address=0.0.0.0:8443"
         - "--upstream=http://127.0.0.1:8080/"
         - "--logtostderr=true"
-        - "--v=10"
+        - "--v=0"
```

Comment:
Is this the log level setting?

Reply:
yes - https://sdk.operatorframework.io/docs/upgrading-sdk-version/v1.16.0/#for-golang-based-projects-add-the-flag-ignore-not-found-into-the-makefile-for-the-uninstall-and-undeploy-targets

Reply:
https://sdk.operatorframework.io/docs/upgrading-sdk-version/v1.17.0/#reduce-debug-log-level-for-the-sidecar-container-kube-rbac-proxy-from-10-to-0